content/php-client/resources/catalog-structure/category-media-files.md

```diff
@@ -0,0 +1,12 @@
+### Category media file
+
+#### Download media file 
+::: php-client-availability versions=11.2.0
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$mediaFile = $client->getCategoryMediaFileApi()->download('code/example');
```

Comment:
I think that we can give a more detailed example on how to get the file_path to download an image since this is not straighforward. Something like this:

```suggestion
// Get categories with attribute values
$firstPage = $client->getCategoryApi()->listPerPage(50, true, ['with_enriched_attributes' => true]);

foreach ($firstPage->getItems() as $category) {
    foreach ($category['values'] as $value) {
        if ($value['type'] === 'image') {
            $filePath = $value['data']['file_path'];
        
            // Download image file for attribute of type image
            $mediaFile = $client->getCategoryMediaFileApi()->download($filePath);
        }
}
```

Reply:
I did it, but with keeping my example at first (for simpler use case).

This is what it's look like now :
![image](https://github.com/akeneo/pim-api-docs/assets/114410848/2bce0cac-5264-4f88-9b88-743e96d4f856)